src/test/java/eu/europa/ted/efx/sdk1/EfxTemplateTranslatorV1Test.java

```diff
@@ -180,6 +180,13 @@ void testStandardLabelReference_UsingLabelTypeAsAssetId() {
         translateTemplate("{BT-00-Text}  #{auxiliary|text|value}"));
   }
 
+  @Test
+  void testStandardLabelReference_WithAssetIdIterator() {
+    assertEquals(
+        "let block01() -> { label(distinct-values(for $item in for $t in ./normalize-space(text()) return $t return concat('field', '|', 'name', '|', $item))) }\nfor-each(/*/PathNode/TextField).call(block01())",
```

Comment:
Having 2 "for" in `for $item in for $t in ./normalize...` looks a bit weird, but it's not a real problem, and I guess it might be unavoidable.

Reply:
yes it look weird but you will not necessarily have a for loop in the sequence expression. The sequence expression might be something like ${('a', 'b', 'c')} for example.